package.json

```diff
@@ -20,6 +20,7 @@
     "react": "^16.10.1",
     "react-dom": "^16.10.1",
     "react-i18next": "^10.13.1",
+    "react-intl": "^3.4.0",
```

Comment:
In this project, we're using `react-i18next`, so we don't need this one.

Reply:
removed

Reply:
Can you remove it also from the packge.json?